Microsoft.DotNet.Cli.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
```

Comment:
Somewhat concerned about all the changes that happened on this file. Don't really understand why they happened.